sdk/appconfiguration/app-configuration/package.json

```diff
@@ -39,11 +39,11 @@
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && dev-tool run bundle --browser-test=false",
+    "build:browser": "tsc -p . && dev-tool run bundle --browser-test=false",
```

Comment:
so in these cases, we're building for the browser, but not bundling to actually test in it?

Reply:
In this case, appconfig has a custom browser configuration. App config can delete its custom config if the issues with the test code can be addressed (making it possible to use the default config).